openproblems/tasks/_batch_integration/batch_integration_graph/README.md

```diff
@@ -36,7 +36,8 @@ Datasets should contain the following attributes:
 * `adata.obsm['X_uni']` with the PCA embedding of the unintegrated representation
 * `adata.obsp['uni_connectivities']` with an unintegrated connectivity matrix generated
   by  `scanpy.pp.neighbors()`
-* `adata.X` with log-normalized data
+* `adata.X` WARNING: other than most tasks, this should contain log-normalized data
```

Comment:
Please explain why

Reply:
I would explaint his in a separate sentence/paragraph, rather than in the data API definition.